common-services.yml

```diff
@@ -16,7 +16,7 @@ services:
       - ./shogun-geoserver/keystore/cacerts:/etc/ssl/certs/java/cacerts
   shogun-postgis:
     container_name: ${CONTAINER_NAME_PREFIX}-postgis
-    image: postgis/postgis:13-3.3-alpine
+    image: postgis/postgis:13-3.4-alpine
```

Comment:
Is Flyway >=9 used in `shogun-boot`? If yes, we could bump postgresql db to at least v15

Reply:
Good point, we're using Flyway in version 9.16.3 in shogun, so this could be updated as well.

I'll check and prepare this in a follow up PR.